pkg/registry/serving/serving.go

```diff
@@ -71,7 +55,7 @@ func (s ServingController) List(namespace string, opt *paging.ListOption) (*Serv
 	sort.SliceStable(sdeps.Items, func(i, j int) bool {
 		return sdeps.Items[i].Name < sdeps.Items[j].Name
 	})
-	return toServingList(sdeps.Items, opt), nil
+	return paging.Page(sdeps.Items, opt), nil
```

Comment:
The  items's type of result struct  is interface{} ，maybe there will can not generate API doc structure, can try it.

Reply:
Yes, but the item's struct can be seen in `GetFunction`, so I guess it's acceptable?

Reply:
If it is ok, LGMT